config/config.go

```diff
@@ -281,6 +281,33 @@ type DeploymentConfig struct {
 // defaults to the namespace configured for IstioNamespace (which itself defaults to 'istio-system').
 type IstioComponentNamespaces map[string]string
 
+// Threshold configuration
+type ThresholdHealth struct {
+	Namespace       string           `yaml:"namespace"`
+	ThresholdChecks []ThresholdCheck `yaml:"threshold_checks"`
+}
+
+type ThresholdCheck struct {
+	Rule       string          `yaml:"rule"`
+	Kind       string          `yaml:"kind"`
```

Comment:
About naming, "Kind" is a very generic word so I don't like it very much but I fail to find something better :-) If anyone has a good idea ...